src/Sylius/Behat/Context/Api/Admin/LoginContext.php

```diff
@@ -89,6 +94,33 @@ public function iResetIt(): void
         $this->client->executeCustomRequest($this->request);
     }
 
+    /**
+     * @When /^(I) follow the instructions to reset my password$/
+     */
+    public function iFollowTheInstructionsToResetMyPassword(AdminUserInterface $admin): void
+    {
+        $this->request = $this->requestFactory->custom(
+            sprintf('%s/admin/reset-password-requests/%s', $this->apiUrlPrefix, $admin->getPasswordResetToken()),
+            HttpRequest::METHOD_PATCH,
+        );
+    }
+
+    /**
+     * @When I specify my new password as :password
+     */
+    public function iSpecifyMyNewPassword(?string $password = null): void
```

Comment:
with the current step specification, a password always will be provided.
```suggestion
    public function iSpecifyMyNewPassword(string $password): void
```